src/cmd/tools/m3ctl/main/client/checker.go

```diff
@@ -0,0 +1,21 @@
+package client
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"go.uber.org/zap"
+)
+
+func checkForAndHandleError(url string, resp *http.Response, zl *zap.SugaredLogger) error {
+	zl.Infof("resp.StatusCode:%d:\n", resp.StatusCode)
```

Comment:
none of the M3DB codebase uses zap Sugared logger if it's possible. the logs outputed by non-sugared logger are actually much nicer.

Reply:
I didn't notice!  It leaked in from another project.